apps/server/src/auth/auth.module.ts

```diff
@@ -63,6 +65,21 @@ export class AuthModule {
             }
           },
         },
+
+        {
+          provide: OidcStrategy,
+          inject: [UserService],
+          useFactory: async (userService: UserService) => {
+            try {
+              const client = await buildOpenIdClient()
+              return new OidcStrategy(client, userService)
+            }
+            catch {
+              return new DummyStrategy()
```

Comment:
If `buildOpenIdClient` were to fail, what would be the consequences of falling back to a DummyStrategy? Does the error get proper visibility?